contracts/network/interfaces/IBancorNetwork.sol

```diff
@@ -273,25 +279,27 @@ interface IBancorNetwork is IUpgradeable {
         uint256 amount,
         IFlashLoanRecipient recipient,
         bytes calldata data
-    ) external;
+    ) external returns (uint256);
 
     /**
      * @dev deposits liquidity during a migration
+     * returns the amount of pool tokens issued
```

Comment:
probably better to use "minted" here as this is what we use all around

Reply:
done